.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "jt12"]
 	path = jt12
 	url = https://github.com/jotego/jt12
+[submodule "jt89"]
+	path = jt89
+	url = git@github.com:jotego/jt89.git
```

Comment:
Better to have https:// url here, git@github.com is for ssh access, and probably only you have ssh access for your own repo.